src/redist/targets/GenerateLayout.targets

```diff
@@ -466,6 +466,12 @@
                       Overwrite="true" />
   </Target>
 
+  <Target Name="LayoutWorkloadUserLocalMarker">
+    <WriteLinesToFile File="$(RedistLayoutPath)metadata/workloads/userlocal"
```

Comment:
In the issue, the version directory was lost when I copied over the information from the PR.
The path should include a version: `/metadata/workloads/<sdkfeatureband>/userlocal`.

The `<sdkfeatureband>` must not include the -suffix. So `6.0.100` (not `6.0.100-rc1`).